packages/nc-gui-v2/composables/useViews.ts

```diff
@@ -0,0 +1,15 @@
+import type { FormType, GalleryType, GridType, KanbanType, TableType } from 'nocodb-sdk'
+import type { Ref } from 'vue'
+import { useNuxtApp } from '#app'
+
+export default function (meta: Ref<TableType>) {
+  const views = ref<Array<GridType | FormType | KanbanType | GalleryType>>()
```

Comment:
You would do me a big favor by using the `[]` notation instead of `Array<>` 😄 

in TS we usually use `string[]` and not `Array<string>` even though both are the exact same.
In this case it would be
```ts
const views = ref<(GridType | FormType | KanbanType | GalleryType)[]>()
```
meaning each element of the array can be a GridType, FormType etc.

or 
```ts
const views = ref<GridType[] | FormType[] ...>()
```

which would mean it's an array of __only__ GridType elements etc.

Reply:
ya sure I will keep it in that format from now onwards.... any specific reason to follow that pattern?

Reply:
No particular reason 😅 its just the usual convention